modin/pandas/test/dataframe/test_reduce.py

```diff
@@ -319,6 +324,10 @@ def test_prod(
     df_equals(modin_result, pandas_result)
 
 
+@pytest.mark.skipif(
+    StorageFormat.get() == "Hdk",
+    reason="https://github.com/intel-ai/hdk/issues/286",
```

Comment:
do we need to skip the whole test? The linked issue mentions only a case of summing an empty array, can we skip just this one case from the test?

Reply:
Xfailed 2 cases from this test.